File: translations/codenet/mistral_commenetd/C2J/gpt-4o/Code/s608809394.java
import java.util.Scanner; // Import the Scanner class for input

public class Main { // Define the main class
    public static void main(String[] args) { // The entry point of the program
        Scanner scanner = new Scanner(System.in); // Create a Scanner object for input

        int n = scanner.nextInt(); // Read an integer value from the standard input and store it in 'n'

        int[] a = new int[n]; // Declare an integer array 'a' of size 'n'

        for (int i = 0; i < n; i++) { // Iterate through each element of the array 'a'
            a[i] = scanner.nextInt(); // Read an integer value from the standard input and store it in the current element of array 'a'
        }

        int t = 0; // Initialize the variable 't' to zero

        for (int i = 0; i < n; i++) { // Iterate through each element of the array 'a'
            int s = 0; // Initialize the variable 's' to zero for summation

            for (int j = 1; j < n; j++) { // Iterate through all other elements of the array 'a' except the current element 'i'
                if (j <= i) { // If the current index 'j' is less than or equal to the current index 'i', then skip the iteration
                    continue;
                }

                int m = a[i] * a[j]; // Multiply the current element 'a[i]' with the next element 'a[j]' and store the result in 'm'

                s = s + m; // Add the product 'm' to the sum 's'
            }

            t = t + s; // Add the sum 's' to the total sum 't'
        }

        System.out.println(t); // Print the total sum 't' to the standard output

        scanner.close(); // Close the scanner
    } // End of main method
} // End of class
// <END-OF-CODE>

File: translations/codenet/mistral_commenetd/C2P/granite/Code/s608809394.py

n = int(input()) # Read an integer value from the standard input and store it in 'n'
a = [int(input()) for _ in range(n)] # Read 'n' integer values from the standard input and store them in the list 'a'

t = 0 # Initialize the variable 't' to zero

for i in range(n): # Iterate through each element of the list 'a'
    s = 0 # Initialize the variable's' to zero for summation

    for j in range(i+1, n): # Iterate through all other elements of the list 'a' except the current element 'i'
        m = a[i] * a[j] # Multiply the current element 'a[i]' with the next element 'a[j]' and store the result in'm'
        s += m # Add the product'm' to the sum's'

    t += s # Add the sum's' to the total sum 't'

print(t) # Print the total sum 't' to the standard output

